src/api/database/models/TranscriptNameMapping.ts

```diff
@@ -0,0 +1,34 @@
+import {
+    Table,
+    Column,
+    Model,
+    ForeignKey,
+    BelongsTo,
+    PrimaryKey,
+  } from 'sequelize-typescript';
+  import { UUID, STRING } from 'sequelize';
+  import User from './User';
+  import Transcript from "./Transcript";
+
+  @Table({ paranoid: true })
+  class NameMapping extends Model {
+    @PrimaryKey
+    @Column(STRING)
+    handlebarName: string;
+
+    @PrimaryKey
+    @ForeignKey(() => Transcript)
+    @Column(STRING)
+    transcriptId: string;
+  
+    @Column(UUID)
+    @ForeignKey(() => User)
+    userId: string | null;
```

Comment:
it should be non-null. use @AllowNull(false);

Reply:
simply removed null should be sufficient, adding allowNull(false) would cause a column annotation error